.buildkite/pipeline.ml.yml

```diff
@@ -335,6 +337,71 @@
     - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only python/ray/tests/horovod/...
 
 
+
+##### STORAGE REFACTOR
+
+- label: ":steam_locomotive: Train tests and examples"
```

Comment:
I think we also need to add:

* :book: Doc tests and examples (excluding Ray AIR examples)
* :airplane: AIR tests (ray/air)
* :airplane: AIR/ML release smoke tests

GPU:
* :tv: :airplane: AIR GPU tests (ray/air)
* :tv: :steam_locomotive: Train GPU tests

Doctest would also probably fail.

Reply:
Ok sounds good - I'll first take a stab with the current selection but can update later/soon.

Reply:
Got it, should we coordinate with a spreadsheet and shard the fixes?

Reply:
Also, let's update the names of these Train / Train + Tune ones to also be prefixed by "New persistence mode:"